.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+	"files.exclude": {
+		"bird-core": true,
```

Comment:
this is excessive?

Reply:
It prevents these directories from appearing in the VS Code file explorer multiple times (when using the provided `.code-workspace` file), which made the Java extension not work properly.

Reply:
ah right..